examples/README.md

```diff
@@ -0,0 +1,83 @@
+# Examples
+
+## JupyterLab with Elyra
+
+This Workbench image installs JupyterLab and the ODH-Elyra extension.
+
+The main difference between the [upstream Elyra](https://github.com/elyra-ai/elyra) and the [ODH-Elyra fork](https://github.com/opendatahub-io/elyra) is that the fork implements Argo Pipelines support, which is required for executing pipelines in OpenDataHub/OpenShift AI.
+Specifically, the fork already includes the changes from [elyra-ai/elyra #3273](https://github.com/elyra-ai/elyra/pull/3273), which is still pending upstream.
+
+### Design
+
+The workbench is based on a Source-to-Image (S2I) UBI9 Python 3.11 image.
+This means—besides having Python 3.11 installed—that it also has the following
+* Python virtual environment at `/opt/app-root` is activated by default
+* `HOME` directory is set to `/opt/app-root/src`
+* port 8888 is `EXPOSE`D by default
+
+These characteristics are required for OpenDataHub workbenches to function.
+
+#### Integration with OpenDataHub Notebook Controller and Notebook Dashboard
+
+#### OpenDataHub Dashboard
+
+Dashboard automatically populates an environment variable named `NOTEBOOK_ARGS` when starting a container from this image.
+This variable contains configurations that are necessary to integrate with Dashboard regarding launching the Workbench and logging off.
+
+Reference: https://github.com/opendatahub-io/odh-dashboard/blob/95d80a0cccd5053dc0ca372effcdcd8183a0d5b8/frontend/src/api/k8s/notebooks.ts#L143-L149
+
+Furthermore, when configuring a workbench, the default Persistent Volume Claim (PVC) is created and volume is mounted at `/opt/app-root/src` in the workbench container.
+This means that changing the user's `HOME` directory from the expected default is inadvisable.
+It further means that whatever the original content of `/opt/app-root/src` in the image may be, it will be shadowed by the PVC.
+
```

Comment:
forgot to mention culling

Reply:
in fact, I did not forget; culling has to be discussed only in relation with codeserver and rstudio; jupyter interfaces with culler for free by default

@coderabbitai create github issue to document culling with codeserver and rstudio for me please